assignment5.js

```diff
@@ -0,0 +1,25 @@
+var fs = require('fs');
+
+var fileReader = function(fileName) {
+  return new Promise( (resolve, reject) => {
+    fs.readFile(fileName, 'utf8', function(error, data) {
+      console.log(data); 
+      console.log("============");
+
+      if(error)
+        return reject(fileName + 'file not Done');
```

Comment:
Why are you sending anything in reject. If you are not using doing the Error handling.
In other words,
- `reject` should return an Error object.
- Error Handling is important. So, introduce it.